src/_guides/language/evolution.md

```diff
@@ -180,6 +180,18 @@ turn into edit-time analysis errors.
 
 In Dart 2.12, **[Dart FFI][]** graduated from beta to the stable channel.
 
+### Dart 2.13
+
+Dart 2.13 added support for **[type aliases][]**,
```

Comment:
Do we need to specify that support was added for "non-function type aliases" or were the function typedefs not referred as type aliases originally?

Reply:
Yeah, either "added support for non-function type aliases" or perhaps "added complete support for type aliases"?

Reply:
I tried something slightly different:

Dart 2.13 expanded support for type aliases (typedef), which used to work only for function types but now work for any type. 

wdyt?